dlrs/main/classes/OptimizerService.cls

```diff
@@ -168,7 +169,38 @@ global with sharing class OptimizerService {
         notification.Id = NotificationReference.LookupRollupSummaryScheduleItemsFieldDeletion;
         notification.Status = NotificationStatus.Attention;
         notification.Message = 'This version of the tool no longer requires the field dlrs__LookupRollupSummary__c on dlrs__LookupRollupSummaryScheduleItems__c. For optimum configuration and performance please delete it after reading the further information provided.';
-        notification.MoreInfoURL = 'https://github.com/afawcett/declarative-lookup-rollup-summaries/wiki/Attention:-Need-to-delete-unused-field-on-dlrs__LookupRollupSummaryScheduleItems__c';
+        notification.MoreInfoURL = 'https://sfdo-community-sprints.github.io/DLRS-Documentation/Issues/#attention-need-to-delete-unused-field-on-dlrs__lookuprollupsummaryscheduleitems__c';
+        return notification;
+      }
+      return null;
+    }
+  }
+
+  /**
+   * Notifies the user if they have to run the "RollupJob" class to process LookupRollupSummaryScheduleItems__c records
+   */
+  public class LookupRollupSummaryScheduleItemsCount extends OptimizationCheckRunner {
+    public override Notification execute() {
+      //Count() aggregate does not hit Row limit.
+      Integer count = [
+        SELECT COUNT()
+        FROM LookupRollupSummaryScheduleItems__c
+        LIMIT 200000
```

Comment:
if our threshold is `3000`, why is the limit so much higher? If we use 5,000, we don't need to riddle if 200,000 will hit maybe any other limit.